example/src/main/java/ably/io/tutorial/ExampleActivity.java

```diff
@@ -26,10 +32,47 @@ protected void onCreate(Bundle savedInstanceState) {
         } catch (AblyException e) {
             e.printStackTrace();
         }
+
+        /* set a button click listener for publishing messages */
+        findViewById(R.id.btPublish).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                /* Always do network instructions outside the Main Thread */
+                new Thread(new Runnable() {
+                    @Override
+                    public void run() {
+                        try {
+                            /* Wrap method in try/catch block to prevent application from crashing due to AblyException */
+                            publishMessages();
+                        } catch (AblyException e) {
+                            e.printStackTrace();
+                        }
+                    }
+                }).start();
+
+            }
+        });
     }
 
     /* Add AblyException to method signature as AblyRest constructor can throw one */
     private void initAbly() throws AblyException {
         AblyRest ablyRest = new AblyRest(API_KEY);
+        /* Get channel for storing sounds */
+        channel = ablyRest.channels.get("persisted:sounds");
     }
+
+    private void publishMessages() throws AblyException {
+        Log.v(getClass().getCanonicalName(), "Publishing three messages...");
+
+        /* Get a null check in case ExampleActivity#initAbly method threw an exception */
+        if (channel != null) {
+            /* Publish three messages, specify event name first, then payload */
+            channel.publish("play", "bark");
+            channel.publish("play", "meow");
+            channel.publish("play", "cluck");
+
+            Log.v(getClass().getCanonicalName(), "Messages sent");
```

Comment:
As with the other PR, will the user even see these messages?  If not, should we not output this to a UI?